<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MBookEdit</class>
 <widget class="QWidget" name="MBookEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>book</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lblIdB">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string notr="true">ID</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>position</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="txtNameB"/>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spnPosBook">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>274</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QPushButton" name="btRefreshB">
           <property name="text">
            <string>refresh</string>
           </property>
           <property name="icon">
            <iconset resource="resfile.qrc">
             <normaloff>:/new/icons/icons/refresh.png</normaloff>:/new/icons/icons/refresh.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btWriteB">
           <property name="text">
            <string>update</string>
           </property>
           <property name="icon">
            <iconset resource="resfile.qrc">
             <normaloff>:/new/icons/icons/save.png</normaloff>:/new/icons/icons/save.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>collection</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lblIdC">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string notr="true">ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtNameC"/>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>format</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cmbInput"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string notr="true">index</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cmbIndex"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>139</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="btRefreshC">
           <property name="text">
            <string>refresh</string>
           </property>
           <property name="icon">
            <iconset resource="resfile.qrc">
             <normaloff>:/new/icons/icons/refresh.png</normaloff>:/new/icons/icons/refresh.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btUpdateC">
           <property name="text">
            <string>update</string>
           </property>
           <property name="icon">
            <iconset resource="resfile.qrc">
             <normaloff>:/new/icons/icons/save.png</normaloff>:/new/icons/icons/save.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>language</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="lblIdLang">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string notr="true">ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtLang"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>274</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QPushButton" name="btRefreshL">
           <property name="text">
            <string>refresh</string>
           </property>
           <property name="icon">
            <iconset resource="resfile.qrc">
             <normaloff>:/new/icons/icons/refresh.png</normaloff>:/new/icons/icons/refresh.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btUpdateL">
           <property name="text">
            <string>update</string>
           </property>
           <property name="icon">
            <iconset resource="resfile.qrc">
             <normaloff>:/new/icons/icons/save.png</normaloff>:/new/icons/icons/save.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>chapter</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spnChapter">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="lblCh">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string notr="true">0-0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="btInsCh">
          <property name="text">
           <string>ins</string>
          </property>
          <property name="icon">
           <iconset resource="resfile.qrc">
            <normaloff>:/new/icons/icons/add_item.png</normaloff>:/new/icons/icons/add_item.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="btAppCh">
          <property name="text">
           <string>app</string>
          </property>
          <property name="icon">
           <iconset resource="resfile.qrc">
            <normaloff>:/new/icons/icons/add_item2.png</normaloff>:/new/icons/icons/add_item2.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QPushButton" name="btDelCh">
          <property name="text">
           <string>del</string>
          </property>
          <property name="icon">
           <iconset resource="resfile.qrc">
            <normaloff>:/new/icons/icons/stop.png</normaloff>:/new/icons/icons/stop.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>32</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="7">
         <widget class="QPushButton" name="btZeroC">
          <property name="text">
           <string>chapter 0</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>verse</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spnVerse">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="lblVe">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string notr="true">0-0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="btInsV">
          <property name="text">
           <string>ins</string>
          </property>
          <property name="icon">
           <iconset resource="resfile.qrc">
            <normaloff>:/new/icons/icons/add_item.png</normaloff>:/new/icons/icons/add_item.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="btAppV">
          <property name="text">
           <string>app</string>
          </property>
          <property name="icon">
           <iconset resource="resfile.qrc">
            <normaloff>:/new/icons/icons/add_item2.png</normaloff>:/new/icons/icons/add_item2.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QPushButton" name="btDelV">
          <property name="text">
           <string>del</string>
          </property>
          <property name="icon">
           <iconset resource="resfile.qrc">
            <normaloff>:/new/icons/icons/stop.png</normaloff>:/new/icons/icons/stop.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>32</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="7">
         <widget class="QPushButton" name="btZeroV">
          <property name="text">
           <string>verse 0</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>112</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QPushButton" name="btRefreshV">
          <property name="text">
           <string>refresh</string>
          </property>
          <property name="icon">
           <iconset resource="resfile.qrc">
            <normaloff>:/new/icons/icons/refresh.png</normaloff>:/new/icons/icons/refresh.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btUpdateV">
          <property name="text">
           <string>update</string>
          </property>
          <property name="icon">
           <iconset resource="resfile.qrc">
            <normaloff>:/new/icons/icons/save.png</normaloff>:/new/icons/icons/save.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QStackedWidget" name="stwKeyb">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QComboBox" name="cmbUtf1">
               <property name="editable">
                <bool>true</bool>
               </property>
               <property name="insertPolicy">
                <enum>QComboBox::NoInsert</enum>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QComboBox::AdjustToContents</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cmbUtf2">
               <property name="editable">
                <bool>true</bool>
               </property>
               <property name="insertPolicy">
                <enum>QComboBox::NoInsert</enum>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QComboBox::AdjustToContents</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btInsUtf">
               <property name="toolTip">
                <string>insert character</string>
               </property>
               <property name="text">
                <string>insert character</string>
               </property>
               <property name="icon">
                <iconset resource="resfile.qrc">
                 <normaloff>:/new/icons/icons/add_item.png</normaloff>:/new/icons/icons/add_item.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_2">
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QComboBox" name="cmbBeta">
               <property name="editable">
                <bool>true</bool>
               </property>
               <property name="insertPolicy">
                <enum>QComboBox::NoInsert</enum>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QComboBox::AdjustToContents</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btInsBeta">
               <property name="toolTip">
                <string>insert character</string>
               </property>
               <property name="text">
                <string>insert character</string>
               </property>
               <property name="icon">
                <iconset resource="resfile.qrc">
                 <normaloff>:/new/icons/icons/add_item.png</normaloff>:/new/icons/icons/add_item.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QFontComboBox" name="cmbfEdit"/>
         </item>
         <item>
          <widget class="QSpinBox" name="spnfEdit">
           <property name="value">
            <number>15</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextBrowser" name="txtEdit">
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
         <property name="openLinks">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QFontComboBox" name="cmbfPrev"/>
         </item>
         <item>
          <widget class="QSpinBox" name="spnfPrev">
           <property name="value">
            <number>15</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextBrowser" name="txtPreview">
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
         <property name="openLinks">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0"/>
 <resources>
  <include location="resfile.qrc"/>
 </resources>
 <connections/>
</ui>
